clusterloader2/testing/load/config.yaml

```diff
@@ -50,8 +54,8 @@ steps:
   - namespaceRange:
```

Comment:
for consistency with density, maybe it should be "group = load" (I mean the label above)?

Reply:
Done.